pages/date-range-picker/month-calendar-permutations.page.tsx

```diff
@@ -26,18 +26,34 @@ const intervals = [
 ];
 
 const permutations = createPermutations<DateRangePickerCalendarProps>([
+  // Selection range
```

Comment:
Personally I'd create new pages instead of modifying new ones, to make the transition smoother. But I'll leave this up to you

Reply:
I think there is a benefit of having those on the same page as it makes it easier to compare the output and ensure the set is complete. I don't have a strong preference either.